docs/index.rst

```diff
@@ -0,0 +1,40 @@
+.. nuclio documentation master file, created by
+   sphinx-quickstart on Wed Sep 20 10:10:53 2023.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
+Welcome to Nuclio documentation!
+==================================
+
+.. toctree::
+   :maxdepth: 2
+
+   concepts/index
+
+.. toctree::
+   :maxdepth: 2
+
+   devel/index
+
+.. toctree::
+   :maxdepth: 2
+
+   reference/index
+
+.. toctree::
+   :maxdepth: 2
+
+   setup/index
+
+
+.. toctree::
+   :maxdepth: 2
+
+   tasks/index
```

Comment:
Change order to:
- Setup
- Concepts
- Tasks -> Change to User Guide
- References
- Development

This order should also be reflected in the sidebar

Reply:
Changed